backend/package.json

```diff
@@ -134,6 +134,7 @@
     "posthog-node": "^3.6.2",
     "probot": "^13.0.0",
     "smee-client": "^2.0.0",
+    "tedious": "^18.2.1",
```

Comment:
got confused at first as to where we're using this but seems like its needed by knex for connections to mssql?

might be worth adding a note somewhere for this implicit dependency